iogt_content_migration/management/commands/load_v1_db.py

```diff
@@ -107,11 +117,13 @@ def migrate(self, root):
         self.migrate_images()
         self.load_page_translation_map()
         home = self.create_home_page(root)
-        section_index_page, banner_index_page, footer_index_page = self.create_index_pages(home)
+        section_index_page, banner_index_page, footer_index_page, poll_index_page, survey_index_page, quiz_index_page = self.create_index_pages(home)
```

Comment:
should we have the `create_index_pages` return a dictionary?

Reply:
done